src/autocomplete.mjs

```diff
@@ -243,14 +250,10 @@ export default class extends Controller {
   }
 
   get src() {
-    return this.data.get("url")
```

Comment:
I sincerely appreciate you taking the time to upgrade the method for configuration.

I'll need to remember to cut a major release/ or a minor release with capital letters to notify the change. Probably a major release since the change to `url` will break everyone.

I am sorry for not considering this -- and considering the general stimulus v1 nature of the plugin -- before asking you to upgrade your own new attribute.

It's probably for the best to just cut a major version and announce upgrade to stimulus 2 values for all configuration.

Reply:
Yes, makes sense.
I've also removed the usage of this getter and replaced by `this.urlValue` directly.